setup.py

```diff
@@ -33,6 +33,11 @@
     )
 
 
+qasm3_import_extras = [
+    # Pinned exactly, because we're including the optional effectively as an alternative to
+    # vendoring the package's code into Terra.
+    "qiskit-qasm3-import==0.1.0",
```

Comment:
As we talked about offline I think this would be better as >=0.1.0 so that we're not stuck needing to push a bugfix release to fix a potential issue in this upstream.

Reply:
Done in e5b409a1a.